testsuite/features/step_definitions/command_steps.rb

```diff
@@ -5,6 +5,27 @@
 require 'timeout'
 require 'nokogiri'
 
+# Sanity checks
+
+Then(/^"([^"]*)" should have a FQDN$/) do |host|
+  node = get_target(host)
+  result, return_code = node.run('hostname -f')
+  raise 'cannot determine hostname' unless return_code.zero?
+  raise 'hostname is not fully qualified' unless result.delete!("\n") == node.full_hostname
+end
+
+Then(/^"([^"]*)" should communicate with the server$/) do |host|
+  node = get_target(host)
+  node.run("ping -c1 #{$server.full_hostname}")
+  $server.run("ping -c1 #{node.full_hostname}")
+end
```

Comment:
It might be good enough, but it will not hurt to instead use netcat to check at same time the port?

Reply:
There's no port for ping (it's ICMP).

Reply:
Yep, I'm saying to replace the ping by a nc that tries to connect to the open port of the server

Reply:
which port ?